indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -710,8 +736,8 @@ private boolean announceTask(
         statusLock.wait(waitMs);
         long elapsed = timeoutStopwatch.elapsed(TimeUnit.MILLISECONDS);
         if (elapsed >= waitMs) {
-          log.error(
-              "Something went wrong! [%s] never ran task [%s]! Timeout: (%s >= %s)!",
+          log.makeAlert(
+              "Something went wrong with worker [%s], never ran task [%s]! Timeout: (%s >= %s)!",
```

Comment:
hmmm.. I don't think its that bad, most of the alert msgs are free flowing text, changing it to suggested string does not seem to be buying much. That is, changing it to `Task assignment timed out for worker[%s]` is not that much different.
However, I changed it a bit for clarity.